components/pkg-export-docker/src/main.rs

```diff
@@ -59,6 +68,8 @@ fn start(ui: &mut UI) -> Result<()> {
     debug!("clap cli args: {:?}", m);
     let default_channel = channel::default();
     let default_url = hurl::default_bldr_url();
+    let registry_type = m.value_of("REGISTRY_TYPE").unwrap_or("docker");
+    let registry_url = m.value_of("REGISTRY_URL");
```

Comment:
What happens if this is left blank?  It doesn't look like you've annotated this parameter as required in the clap macro, so you should either do that, or provide a default value (perhaps of the public docker registry, since the default registry type is docker).

Reply:
The docker client hard-codes the value for the default registry so I actually want a `Some` or `None` here.